### PR TITLE
[Rule Tuning] Component Object Model Hijacking

### PR DESCRIPTION
**Pull request by LaZyDK**
## Summary
Add HKEY_USERS\*Classes\CLSID\*\LocalServer32\ to exclusions.

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/persistence_suspicious_com_hijack_registry.toml [hunk 1]
```diff
@@ -32,7 +32,8 @@ registry where
  or
  /* in general COM Registry changes on Users Hive is less noisy and worth alerting */
  (registry.path : ("HKEY_USERS\\*Classes\\*\\InprocServer32\\",
-                   "HKEY_USERS\\*Classes\\*\\LocalServer32\\", 
+                   "HKEY_USERS\\*Classes\\*\\LocalServer32\\",
```
Comment (Samirbous) [inline rules/windows/persistence_suspicious_com_hijack_registry.toml:35]: @LaZyDK  "`HKEY_USERS\\*Classes\\*\\LocalServer32\\`" already covers also "`HKEY_USERS\\*Classes\\CLSID\\*\\LocalServer32\\`" by way of wildcard, or I'm missing something ?

Comment (LaZyDK) [reply rules/windows/persistence_suspicious_com_hijack_registry.toml:35]: The rule still triggers on this document:
```{
  "@timestamp": "2021-09-18T17:24:49.928Z",
  "data_stream": {
    "dataset": "endpoint.events.registry",
    "namespace": "namespace",
    "type": "logs"
  },
  "ecs": {
    "version": "1.6.0"
  },
  "event": {
    "action": "modification",
    "category": "registry",
    "created": "2021-09-18T17:21:42.084Z",
    "dataset": "endpoint.events.registry",
    "ingested": "2021-09-18T17:21:54.093Z",
    "kind": "signal",
    "module": "endpoint",
    "sequence": "108862",
    "type": "change"
  },
  "message": "Endpoint registry event",
  "process": {
    "executable": "C:\\Program Files\\Veeam\\Backup and Replication\\Console\\veeam.backup.shell.exe",
    },
    "name": "veeam.backup.shell.exe",
    "pid": "9212"
  },
  "registry": {
    "data": {
      "strings": "\"C:\\Program Files\\Veeam\\Backup and Replication\\Console\\veeam.backup.shell.exe\" -ToastActivated",
      "type": "REG_SZ"
    },
    "hive": "HKEY_USERS",
    "key": "S-1-5-21-11469200-209135752-285429281-8777_Classes\\CLSID\\{0a3fb2d4-8a6d-4678-8fdf-e4a78f689d41}\\LocalServer32",
    "path": "HKEY_USERS\\S-1-5-21-11469200-209135752-285429281-8777_Classes\\CLSID\\{0a3fb2d4-8a6d-4678-8fdf-e4a78f689d41}\\LocalServer32\\",
    "value": ""
  },
  "user": {
    "domain": "Domain",
    "id": "S-1-5-21-00000000-209135752-0000000-8777",
    "name": "BackupUser"
  }
}```

Comment (Samirbous) [reply rules/windows/persistence_suspicious_com_hijack_registry.toml:35]: then its better to tune by the couple process.executable and registry.path : 

`and not (process.executable :  "C:\\Program Files\\Veeam\\Backup and Replication\\Console\\veeam.backup.shell.exe" and registry.path : "HKEY_USERS\\S-1-5-21-*_Classes\\CLSID\\{0a3fb2d4-8a6d-4678-8fdf-e4a78f689d41}\\LocalServer32\\")`

Comment (LaZyDK) [reply rules/windows/persistence_suspicious_com_hijack_registry.toml:35]: Woudn't it be best for everybody to make it more generic, and not Veeam specific by just adding `"HKEY_USERS\\*Classes\\CLSID\\*\\LocalServer32\\"`?

Comment (w0rk3r) [reply rules/windows/persistence_suspicious_com_hijack_registry.toml:35]: @Samirbous how can we proceed on this one?

Comment (Samirbous) [reply rules/windows/persistence_suspicious_com_hijack_registry.toml:35]: as reported here https://github.com/elastic/detection-rules/pull/1491#discussion_r713816565 we can apply following tuning:
```
and not (process.executable : "C:\\Program Files\\Veeam\\Backup and Replication\\Console\\veeam.backup.shell.exe" and registry.path : "HKEY_USERS\\S-1-5-21-*_Classes\\CLSID\\*\\LocalServer32\\")
```

File: rules/windows/persistence_suspicious_com_hijack_registry.toml [hunk 2]
```diff
@@ -32,7 +32,8 @@ registry where
...
+                   "HKEY_USERS\\*Classes\\CLSID\\*\\LocalServer32\\",
                    "HKEY_USERS\\*Classes\\*\\DelegateExecute\\", 
                    "HKEY_USERS\\*Classes\\*\\TreatAs\\", 
                    "HKEY_USERS\\*Classes\\CLSID\\*\\ScriptletURL\\") and
```
Comment (w0rk3r) [inline rules/windows/persistence_suspicious_com_hijack_registry.toml:39]:
```suggestion
                   "HKEY_USERS\\*Classes\\CLSID\\*\\ScriptletURL\\") and
 not (process.executable : "C:\\Program Files\\Veeam\\Backup and Replication\\Console\\veeam.backup.shell.exe" and
      registry.path : "HKEY_USERS\\S-1-5-21-*_Classes\\CLSID\\*\\LocalServer32\\") and
```

**Review by w0rk3r — CHANGES_REQUESTED** [review]

File: rules/windows/persistence_suspicious_com_hijack_registry.toml [hunk 3]
```diff
@@ -32,7 +32,8 @@ registry where
  or
  /* in general COM Registry changes on Users Hive is less noisy and worth alerting */
  (registry.path : ("HKEY_USERS\\*Classes\\*\\InprocServer32\\",
-                   "HKEY_USERS\\*Classes\\*\\LocalServer32\\", 
+                   "HKEY_USERS\\*Classes\\*\\LocalServer32\\",
+                   "HKEY_USERS\\*Classes\\CLSID\\*\\LocalServer32\\",
```
Comment (w0rk3r) [inline rules/windows/persistence_suspicious_com_hijack_registry.toml:36]:
```suggestion
```

**Review by w0rk3r — CHANGES_REQUESTED** [review]

**Review by Samirbous — APPROVED** [review]
👍

**Review by w0rk3r — APPROVED** [review]

**Pull request merged**